at_client/lib/src/service/sync_service_impl.dart

```diff
@@ -445,16 +474,19 @@ class SyncServiceImpl implements SyncService, AtSignChangeListener {
     if (switchAtSignEvent.previousAtClient?.getCurrentAtSign() ==
         _atClient.getCurrentAtSign()) {
       // actions for previous atSign
+      _syncRequests.clear();
       _logger.finer(
           'stopping stats notificationlistener for ${_atClient.getCurrentAtSign()}');
       _statsNotificationListener.stopAllSubscriptions();
+      _cron.close();
+      _syncServiceMap.remove(_atClient.getCurrentAtSign());
     }
```

Comment:
You need an else here to start the cron. @X is switched to @Y and switched backed to @X is the scenario.

Reply:
i am clearing the sync service map ..when y is switched back to x..cron and stats listener will be started as a part of create

Reply:
Ok. Let's test the scenario once see if it is happening fine.

Reply:
I still think you need an else to start the cron again